azure-communication-ui/build.gradle

```diff
@@ -51,11 +51,16 @@ buildscript {
             return rootProject.hasProperty("disableTaskRootCheck") && rootProject.getProperty("disableTaskRootCheck")
         }
 
-        compose_version = '1.3.1'
+        androidx_lifecycle_runtime_ktx_version = '2.3.1'
+        androidx_activity_compose_version = '1.5.1'
+        compose_version = '1.2.1'
+        compose_material3_version = '1.0.0-alpha12'
+        kotlin_compiler_extension_version = '1.3.1'
     }
 
     repositories {
         google()
+        mavenCentral()
```

Comment:
would it be possible to know the cause for adding maven here?

Reply:
This was added by mistake. Mixed up this repo block with allrepo block. Removing this in next commit.